rating-submit.js

```diff
@@ -0,0 +1,242 @@
+import { LitElement, html, css } from "https://unpkg.com/lit-element/lit-element.js?module";
+//  import event
+export class RatingModal extends LitElement {
```

Comment:
Please name it SubmitRating in place of RatingModal

Reply:
Done